bundles/org.openhab.binding.samsungtv/src/main/java/org/openhab/binding/samsungtv/internal/WakeOnLanUtility.java

```diff
@@ -102,30 +116,33 @@ public class WakeOnLanUtility {
     /**
      * Send single WOL (Wake On Lan) package on all interfaces
      *
-     * @param macAddress MAC address to send WOL package to
+     * @macAddress MAC address to send WOL package to
      */
+    @SuppressWarnings("null")
```

Comment:
Could you try to get rid of this suppression and fix the warning in the code? 
If you introduce a false positive warning due to null annotation incompatibility, that is ok. Better to show that then to hide potential issues.